client/src/components/pages/ViewBuildings/Map.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet';
+import PropTypes from 'prop-types';
+
+import styles from './view.module.css';
+
+const MapComponent = props => {
+  const { buildingInfo } = props;
+  const markerBuild = () => {
+    const positions = buildingInfo.map(building => {
```

Comment:
the values you return don't indicate that the object is represinting positions only